plat/common/lcpu.c

```diff
@@ -404,8 +404,9 @@ void __weak __noreturn lcpu_entry_default(struct lcpu *this_lcpu)
 	}
 }
 
-int ukplat_lcpu_start(const __lcpuidx lcpuidx[], unsigned int *num, void *sp[],
-		      const ukplat_lcpu_entry_t entry[], unsigned long flags)
+int ukplat_lcpu_start(const __lcpuidx lcpuidx[], unsigned int *num, void *pt_base[],
```

Comment:
I think having to pass in the raw pointer to the base of the page table is a bit unclean. However, I'm not sure whether there is a better way without having to require paging support. (Maybe just default to the current/static page table if the pointer is `NULL`?)
Also using `__paddr_t` would be the better type for the parameter.